src/app/ui-library/inputFeedback.tsx

```diff
@@ -0,0 +1,50 @@
+import { createElement } from 'react';
+import { CheckCircle, Info, Warning, WarningCircle } from '@phosphor-icons/react';
+import styled from 'styled-components';
+import Theme from 'theme';
+import { StyledP } from './common.styled';
+
+type FeedbackVariant = 'info' | 'danger';
+
+const colors = {
+  info: Theme.colors.white_200,
+  danger: Theme.colors.danger_light,
+};
+const getColorForVariant = (variant: FeedbackVariant) => colors[variant];
+
+const icons = {
+  info: Info,
+  warning: WarningCircle,
+  success: CheckCircle,
```

Comment:
Do we need these if the variants on feedback don't support them?

Reply:
good catch, thanks. initially I had 4 variants and then decided to not preempt design 😝